ScoutSuite/providers/azure/resources/sql/server_azure_ad_administrators.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from azure.mgmt.sql import SqlManagementClient
+from msrestazure.azure_exceptions import CloudError
+
+from ScoutSuite.providers.base.configs.resources import Resources
+
+
+class ServerAzureAdAdministrators(Resources):
+
+    def __init__(self, resource_group_name, server_name, facade):
+        self.resource_group_name = resource_group_name
+        self.server_name = server_name
+        self.facade = facade
+
+    # TODO: make it really async.
+    async def fetch_all(self):
+        try:
+            _ = self.facade.server_azure_ad_administrators.get(self.resource_group_name, self.server_name)
```

Comment:
Do you need to store the result in a variable?

Reply:
Haha, nice catch.